frontend/src/components/gotEnzymes/Landing.vue

```diff
@@ -0,0 +1,183 @@
+              type="text"
+              placeholder="search"
+              @keyup.enter="updateSearch()"
+              @input="searchStringChange()"
+            />
+            <span
+              v-show="showSearchCharAlert"
+              class="has-text-info icon is-right"
+              style="width: 220px"
+            >
+              Type at least 2 characters
+            </span>
+            <span class="icon is-medium is-left">
+              <i class="fa fa-search is-primary"></i>
+            </span>
+          </p>
+          <div v-if="searchResultsEmpty" class="mt-2">
+            <div v-if="searchTerm" class="has-text-centered notification">
+              {{ messages.searchNoResult }} for
+              <b>
+                <i>{{ searchTerm }}</i> </b
+              >. Please search using a valid EC code or KEGG id for reaction or compound.
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="columns is-centered pb-6">
+      <div class="column is-three-fifths-desktop is-three-quarters-tablet is-fullwidth-mobile">
+        <div class="content">
+          <h5 class="title is-5">You can search for a:</h5>
+          <ul>
+            <li>
+              reaction (e.g.
+              <router-link to="/gotEnzymes/reaction/R01234"> R01234 </router-link>
+              )
+            </li>
+            <li>
+              EC code (e.g.
+              <router-link to="/gotEnzymes/ec/2.5.1.19">2.5.1.19</router-link>
+              )
+            </li>
+            <li>
+              compound (e.g.
+              <router-link to="/gotEnzymes/compound/C00003">C00003</router-link>
+              )
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+
+    <section class="section extended-section">
+      <div class="container is-fullhd">
+        <h3 class="title is-3">About the project</h3>
+        <div class="columns is-variable is-8 pt-5">
+          <TableOfContents :links="tocLinks" />
+          <div id="intro" class="column content has-text-justified">
+            <p>
+              GotEnzymes provides open access to over XYZ million predicted kinetic data for XYZ
+              unique sequences and XYZ compounds from XYZ species with the cutting-edge artificial
+              intelligence tools. Data in this database are predicted from available advanced deep
+              learning tools: kcat[reference] and KM [reference]. Our vision for this database is to
+              store every possible predicted parameter for any available enzyme to facilitate better
+              science.
+            </p>
+
+            <hr class="mt-6" />
+            <h4 id="how-to-use-it" class="is-info is-size-4">How to use it</h4>
+            <p>
+              We now support searches by reaction, compound, EC number. Each predicted parameter is
+              filterable. Feel free to contact us if you find any issue. To report a technical error
```

Comment:
Who should you contact if the issue is not technical? :') I think it makes sense to just have `Feel free to contact us if you find any issues at contact@metabolicatlas.org`

Reply:
For the GEMs we have a different contact set up, and I think it would make sense here as well. My suggestion would be to leave this as is, and I will follow up with the researchers.

Reply:
Addressed in https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1031/commits/ccc3b575c69fa169b997f4808d9c85be9ac2e1ca